<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LANChatServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="serverWindowGridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="addressesGroupBox">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string>Адреса сервера</string>
      </property>
      <layout class="QGridLayout" name="addressesGridLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="addressesListWidget">
         <property name="minimumSize">
          <size>
           <width>272</width>
           <height>96</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>272</width>
           <height>96</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="logGroupBox">
      <property name="minimumSize">
       <size>
        <width>471</width>
        <height>581</height>
       </size>
      </property>
      <property name="title">
       <string>Лог</string>
      </property>
      <layout class="QGridLayout" name="chatGridLayout">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="logPlainTextEdit">
         <property name="minimumSize">
          <size>
           <width>445</width>
           <height>528</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="usersGroupBox">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>425</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Пользователи</string>
      </property>
      <layout class="QGridLayout" name="usersGridLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="usersListWidget">
         <property name="minimumSize">
          <size>
           <width>272</width>
           <height>371</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
